nixos/modules/misc/meta.nix

```
@@ -81,8 +139,51 @@ in
         '';
       };
 
+      tests = lib.mkOption {
+        type = testsType;
+        description = ''
+          Tests for particular modules.
+
+          Definitions are tied to the location of the module, so the definition syntax is distinct from the option value format.
+
+          Example definition:
+          ```nix
+          # module.nix
+          {
+            meta.tests = { nixosTests }:
```

Nit: nixfmt would format like this:

```suggestion
            meta.tests =
              { nixosTests }:
```

Should the example be consistent?